pkg/bot/slack_cloud.go

```diff
@@ -600,6 +600,24 @@ func (b *CloudSlack) send(ctx context.Context, event slackMessage, resp interact
 	return nil
 }
 
+func (b *CloudSlack) uploadFileToSlack(ctx context.Context, event slackMessage, resp interactive.CoreMessage) (*slack.File, error) {
```

Comment:
Why can't we use the `uploadFileToSlack` from other Slacks? From what I can see it is 1:1 - instead of copying the logic you could run that function 🤔 

BTW it's in `slack_legacy.go` but we can move it to `slack_shared.go`.

Reply:
this function is using new `event.GetTimestamp()` func so the messages are posted in the thread, while on socket slack it will still land on the channel level 👍